### PR TITLE
Add Competition Entry Page

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@miltontony ready for review

**Comment by miltontony** [comment]
needs some tests and stuff.. I say continue working on this until you atleast have the basic templates

File: molo/yourwords/views.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.views.generic.edit import CreateView
+from molo.yourwords.forms import CompetitionEntryForm
+
+
+class CompetitionEntry(CreateView):
+    form_class = CompetitionEntryForm
+    template_name = 'yourwords/Your_words_competition.html'
```
Comment (miltontony) [inline molo/yourwords/views.py]: all lowercase please - just to keep things standard

File: molo/yourwords/views.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django.views.generic.edit import CreateView
+from molo.yourwords.forms import CompetitionEntryForm
+
+
+class CompetitionEntry(CreateView):
```
Comment (miltontony) [inline molo/yourwords/views.py:11]: Also can you call this `CompetitionEntryView`, just to make sure it's clear from the name

File: molo/yourwords/urls.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+from django.conf.urls import patterns, url
+# from django.contrib.auth.decorators import login_required
+
+urlpatterns = patterns(
+    '',
+    url(
+        r'^entry/$',
+        views.CompetitionEntry.as_view(
+            template_name="yourwords/Your_words_competition.html"
```
Comment (miltontony) [inline molo/yourwords/urls.py]: You don't need to specify this if it's specified in the CompetitionEntryView

**Pull request merged**